amazon-eks-nodegroup.yaml

```diff
@@ -507,7 +574,12 @@ Resources:
                      --resource NodeGroup  \
                      --region ${AWS::Region}
         MetadataOptions:
-          "HttpPutResponseHopLimit" : 2
+          HttpPutResponseHopLimit : 2
+          HttpEndpoint: enabled
+          HttpTokens: !If
+            - IMDSv1Disabled
+            - required
+            - optional
```

Comment:
I think this is a bigger change than the added ARM instances in the drop down. 😄